front/src/modules/ui/editable-field/components/EditableField.tsx

```diff
@@ -132,7 +129,7 @@ export function EditableField({
         ) : (
           <EditableFieldDisplayMode
             disableHoverEffect={disableHoverEffect}
-            disableClick={useEditButton}
```

Comment:
We now want to be able to click when there is the editButton, I've added a disableClick props to keep the flexibility